packages/thirdweb/src/wallets/smart/lib/paymaster.ts

```diff
@@ -53,9 +52,7 @@ export async function getPaymasterAndData(args: {
   };
 
   const entrypoint = entrypointAddress ?? ENTRYPOINT_ADDRESS_v0_6;
-  const entrypointVersion = getEntryPointVersion(entrypoint);
-  const paymasterVersion = entrypointVersion === "v0.6" ? "v1" : "v2";
-  const paymasterUrl = getDefaultBundlerUrl(chain, paymasterVersion);
+  const paymasterUrl = getDefaultBundlerUrl(chain);
```

Comment:
there was no reason why we would need to point at `/v1`? `/v2` handles everything `/v1` did?